docs/dml-of-carbondata.md

```diff
@@ -334,11 +335,21 @@ CarbonData DML statements are documented here,which includes:
     OPTIONS('batch_file_count'='5')
     ```
 
+  - ##### BATCH_FILE_ORDER: 
+    The order type of stage files in per processing, choices: ASC, DESC.
+    The default is ASC.
```

Comment:
This is ascending or descending based on filename or filesize or file creation time. better to describe here in comments

Reply:
Down I saw  that, it is file last modified time. better to mention in the document and in comments

Reply:
Done, comments added.